src/main/java/seedu/gamebook/model/gameentry/GameEntry.java

```diff
@@ -180,6 +180,14 @@ public boolean equals(Object other) {
         return false;
     }
 
+    /**
+     * Checks equality in terms of semantic content. A stronger equality checker than {@code equals}.
+     */
+    public boolean isSameContent(GameEntry otherGameEntry) {
+        return this.toString().equals(otherGameEntry.toString());
+    }
+
+
```

Comment:
I think we could just change the equals method of gameEntry instead of doing this? I tried changing `GameEntry#equals` to the following, and it seems to work. I think it might be better to do that instead of having another method which is similar to equals?
```
public boolean equals(Object other) {
    if (this == other) {
        return true;
    } else if (other instanceof GameEntry) {
        GameEntry tmp = (GameEntry) other;
        return gameType.equals(tmp.gameType)
                && getDifference().equals(tmp.getDifference())
                && date.equals(tmp.date)
                && durationMinutes.equals(tmp.durationMinutes)
                && location.equals(tmp.location)
                && tags.equals(tmp.tags);
    }
    return false;
}
```